cli/create_indices.go

```diff
@@ -47,8 +49,22 @@ func (c *createIndices) Run() {
 
 	if exists {
 		c.logger.Println("Person index already exists")
-		c.exit(0)
-		return
+
+		if *c.force {
+			c.logger.Println("Changes are forced, deleting old index")
+			err := c.esClient.DeleteIndex(person.Person{})
+
+			if err != nil {
+				c.logger.Println(err)
+				c.exit(1)
+				return
+			}
+
+			c.logger.Println("Person index deleted successfully")
+		} else {
+			c.exit(0)
+			return
```

Comment:
I won't make you do all the refactors I wanted to do, but wouldn't it be nice to return the exit code and have the `c.exit()` call at the level above